src/backend/access/appendonly/test/aomd_test.c

```diff
@@ -58,6 +60,17 @@ mock_unlink(const char *path)
 #endif
 	return ec;
 }
+
+/*
+ * Make do_truncate return always Ok during unit test,
+ * because otherwise mdunlink_ao will return before actual invoke of unlink,
+ * if called with test file names.
+ */
+int
+__wrap_do_truncate(const char *path)
```

Comment:
add some words about this change to PR desription

Reply:
done